Add rituals section to product page (#5178)
fleetdm/fleet: handbook/product.md

We track the capabilities of competitors and adjacent (or commonly integrated) products in this [Google Doc](https://docs.google.com/document/d/1Bqdui6oQthdv5XtD5l7EZVB-duNRcqVRg7NVA4lCXeI/edit) (private).

## Product rituals

The following rituals are engaged in by the  directly responsible individual (DRI) and at the frequency specified for the ritual.

> ## Brand rituals
> 
> The following rituals are engaged in by the  directly responsible individual (DRI) and at the frequency specified for the ritual.
> 

Reviewer: ```suggestion
```
It looks like this was pasted from a different section of the handbook. I could be totally wrong here.

| Ritual                       | Frequency                | Description                                         | DRI               |
|:-----------------------------|:-----------------------------|:----------------------------------------------------|-------------------|
| Product huddle | Daily | "In progress" issues are discussed and any issues "ready for review" are placed on the list for the product design review call. On Mondays, issues are broken down into a week's work and added into "ready." Issues are moved out of "delivered" every Friday. | Noah Talerman |
| 🗣 Product office hours  | Weekly (Tuesdays) | Decision is made regarding which customer and community feature requests can be committed to in the next six weeks. Issues are created for any requests that don't already have one. | Noah Talerman |
| 🎨UI/UX office hours      | Weekly (Wednesdays) | Decision is made regarding which community UX requests can be committed to in the next six weeks. Community UX questions are discussed, as are questions from the Slack community and internal Fleet team with the intention of providing answers to the question's originator.    | Noah Talerman |

> | ✨Product design review  | Weekly (Thursdays) | Product team discusses "ready for review" items and the decision is made on whether to pass UI changes on to engineering. | Noah Talerman |

Reviewer: ```suggestion
| ✨Product design review  | Weekly (Thursdays) | Product team discusses "ready for review" items and the decision is made on whether the UI changes are ready for engineering specification, and later, implementation. | Noah Talerman |
```
I suggest this small tweak. This way, we indicate that product and engineering collaborate to put the change in place instead of passing a change over a product ->  engineering wall.
